src/test/kotlin/lotto/LottoShopTest.kt

```diff
@@ -0,0 +1,29 @@
+package lotto
+
+import lotto.domain.LottoShop
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.CsvSource
+import org.junit.jupiter.params.provider.ValueSource
+
+class LottoShopTest {
+
+    @ParameterizedTest
+    @ValueSource(strings = ["1", "1001", "2500", "3500"])
+    fun `로또 구매 금액이 천원단위로 들어오는지 테스트한다 천원단위가 아닐시 IllegalArgumentException 에러가 발생한다`(money: Int) {
```

Comment:
```suggestion
    fun `로또 구매 금액이 천원단위가 아닐시 IllegalArgumentException 에러가 발생한다`(money: Int) {
```
"천원단위로 들어오는지 테스트한다" 문구가 없어도 이해할 수 있을 것 같아요